Modules/Quotas/src/Quotas.Domain/Aggregates/Identities/TierQuota.cs

```diff
@@ -18,4 +18,9 @@ public TierQuota(TierQuotaDefinition definition, string applyTo) : base(applyTo)
     public override MetricKey MetricKey => _definition.MetricKey;
     public override int Max => _definition.Max;
     public override QuotaPeriod Period => _definition.Period;
+
+    public TierQuotaDefinitionId GetTierQuotaDefinitionId()
+    {
+        return _definition.Id;
+    }
```

Comment:
Rather add a `DefinitionId` property for it. And remember to exclude it in the EntityTypeConfiguration.